doc/NOW-spec.md

```diff
@@ -761,7 +801,8 @@ The NOW_EXEC_MSG message is used to execute remote commands or scripts.
 
 #### NOW_EXEC_ABORT_MSG
 
-The NOW_EXEC_ABORT_MSG message is used to abort a remote execution immediately due to an unrecoverable error. This message can be sent at any time without an explicit response message. The session is considered aborted as soon as this message is sent.
+The NOW_EXEC_ABORT_MSG message is used to abort a remote execution immediately. See NOW_EXEC_CANCEL_REQ if
+the graceful session cancellation is needed instead. This message can be sent by the client at any point of session lifetime. The session is considered aborted as soon as this message is sent.
```

Comment:
suggestion: I recommend you to use the "sentence-per-line" style: https://github.com/Devolutions/IronRDP/blob/25bbb2682c95419fc18a61cab81e29a0ab9f23d4/STYLE.md#sentence-per-line-style

```suggestion
The NOW_EXEC_ABORT_MSG message is used to abort a remote execution immediately.
See NOW_EXEC_CANCEL_REQ if the graceful session cancellation is needed instead.
This message can be sent by the client at any point of session lifetime.
The session is considered aborted as soon as this message is sent.
```